kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/VaultKmsTestKekManager.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.provider.hashicorp.vault;
+
+import java.net.URI;
+import java.net.http.HttpRequest;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import io.kroxylicious.kms.provider.hashicorp.vault.model.UpdateKeyConfigRequest;
+import io.kroxylicious.kms.service.TestKekManager;
+
+import static io.kroxylicious.kms.provider.hashicorp.vault.VaultKmsTestUtils.createVaultDelete;
+import static io.kroxylicious.kms.provider.hashicorp.vault.VaultKmsTestUtils.createVaultGet;
+import static io.kroxylicious.kms.provider.hashicorp.vault.VaultKmsTestUtils.createVaultPost;
+import static io.kroxylicious.kms.provider.hashicorp.vault.VaultKmsTestUtils.getBody;
+import static io.kroxylicious.kms.provider.hashicorp.vault.VaultKmsTestUtils.sendRequest;
+import static io.kroxylicious.kms.provider.hashicorp.vault.VaultKmsTestUtils.sendRequestExpectingNoContentResponse;
+import static java.net.URLEncoder.encode;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class VaultKmsTestKekManager implements TestKekManager {
```

Comment:
Rather than extracting `VaultKmsTestUtils` I think this class should extend or wrap a `ValutKmsKetManager`